send/o365_mu_process.pl

```diff
@@ -33,6 +33,7 @@
 
 #number of worker threads
 my $THREAD_COUNT = 10;
+our $GLOBAL_RETURN_CODE=0;
```

Comment:
Let's keep this local (my). There is no need to access this variable from other imported files/libraries.